ksql-functional-tests/src/test/resources/query-validation-tests/create-struct.json

```diff
@@ -0,0 +1,20 @@
+{
+  "comments": [
+    "Tests covering the use of the array returning UDFs."
+  ],
+  "tests": [
+    {
+      "name": "entries sorted",
```

Comment:
Can we add a nested struct creation test please?

Reply:
And another that sees what happens if you try to create a struct with no fields...

Reply:
second test is in `insert-values` but I'll add one here too. Good call with nested structs.